packages/glibc_build237.rb

```diff
@@ -0,0 +1,323 @@
+        system "make -j1 DESTDIR=#{CREW_DEST_DIR} install || true" # "sln elf/symlink.list" fails on armv7l
+      when 'i686', 'x86_64'
+        system "make -j1 DESTDIR=#{CREW_DEST_DIR} install || make -j1 DESTDIR=#{CREW_DEST_DIR} install || true"
+      end
+      if @libc_version.to_f >= 2.32
+        system "install -Dt #{CREW_DEST_PREFIX}/bin -m755 build-locale-archive"
+        system "make -j1 DESTDIR=#{CREW_DEST_DIR} localedata/install-locales"
+      end
+      Dir.chdir CREW_DEST_LIB_PREFIX do
+        puts "System glibc version is #{LIBC_VERSION}.".lightblue
+        puts 'Creating symlinks to system glibc version to prevent breakage.'.lightblue
+        @crew_libc_version = @libc_version
+        case ARCH
+        when 'aarch64', 'armv7l'
+          FileUtils.ln_sf "/lib/ld-#{@crew_libc_version}.so", 'ld-linux-armhf.so.3'
+        when 'i686'
+          FileUtils.ln_sf "/lib/ld-#{@crew_libc_version}.so", 'ld-linux-i686.so.2'
+        when 'x86_64'
+          FileUtils.ln_sf "/lib64/ld-#{@crew_libc_version}.so", 'ld-linux-x86-64.so.2'
+        end
+        @libraries = %w[ld libBrokenLocale libSegFault libanl libc libcrypt
+                        libdl libm libmemusage libmvec libnsl libnss_compat libnss_db
+                        libnss_dns libnss_files libnss_hesiod libpcprofile libpthread
+                        libthread_db libresolv librlv librt libthread_db-1.0 libutil]
+        @libraries -= ['libpthread'] if @crew_libc_version.to_f >= 2.35
+        @libraries.each do |lib|
+          # Reject entries which aren't libraries ending in .so, and which aren't files.
+          Dir["/#{ARCH_LIB}/#{lib}.so*"].reject { |f| File.directory?(f) }.each do |f|
+            @filetype = `file #{f}`.chomp
+            if ['shared object', 'symbolic link'].any? { |type| @filetype.include?(type) }
+              g = File.basename(f)
+              FileUtils.ln_sf f.to_s, "#{CREW_DEST_LIB_PREFIX}/#{g}"
+            end
+          end
+          # Reject entries which aren't libraries ending in .so, and which aren't files.
+          # Reject text files such as libc.so because they points to files like
+          # libc_nonshared.a, which are not provided by ChromeOS
+          Dir["/usr/#{ARCH_LIB}/#{lib}.so*"].reject { |f| File.directory?(f) }.each do |f|
+            @filetype = `file #{f}`.chomp
+            puts "f: #{@filetype}" if @opt_verbose
+            if ['shared object', 'symbolic link'].any? { |type| @filetype.include?(type) }
+              g = File.basename(f)
+              FileUtils.ln_sf f.to_s, "#{CREW_DEST_LIB_PREFIX}/#{g}"
+            elsif @opt_verbose
+              puts "#{f} excluded because #{@filetype}"
+            end
+          end
+        end
+      end
+    end
+    # Only save libnsl.so.2, since libnsl.so.1 is provided by perl
+    # For this to work, build on a M107 or newer container.
+    FileUtils.cp File.realpath("#{CREW_DEST_LIB_PREFIX}/libnsl.so.1"), "#{CREW_DEST_LIB_PREFIX}/libnsl.so.2"
```

Comment:
Aren't you already indicating the full path?

Reply:
This. resolves symlinks.

Reply:
Ah, that makes sense.